environ/project.clj

```diff
@@ -1,7 +1,18 @@
-(defproject environ "1.1.0"
+(defproject environ "1.1.1"
```

Comment:
not sure if I should mess with the version, and if it should be a minor update instead?